### ISSUE TITLE
Docker Images fail on `helm diff` when `HOME` env variable is changed

### ISSUE THREAD

**Issue by mptr** [helmfile/helmfile#574]
### Operating system

Ubuntu 20.04.5 LTS

### Helmfile Version

v0.148.1

### Helm Version

version.BuildInfo{Version:"v3.10.2", GitCommit:"50f003e5ee8704ec937a756c646870227d7c8b58", GitTreeState:"clean", GoVersion:"go1.18.8"}

### Bug description

The Bug occours when using the Docker images inside a github action since the action takes control over the HOME environemnt variable. It seems that in this case helm cant find the preinstalled diff extension anymore.

### Example helmfile.yaml

<issue applies to any helmfile>

### Error message you've seen (if any)

Error: unknown command "diff" for "helm"

### Steps to reproduce

```
docker run --rm -e HOME=useless ghcr.io/helmfile/helmfile-debian-stable-slim:v0.148.1 helm diff
```

### Working Helmfile Version

none

**Comment by yxxhero** [comment]
@mptr https://github.com/helmfile/helmfile/pull/546

**Comment by mptr** [comment]
I did not change it myself but [GitHub Actions working with Docker](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action) seem to take control over the environment variable.
I managed to get it to work by just (re)setting the HOME variable back to `/root` in an entrypoint script. This might be a very dirty solution but it seems to work for now. Not breaking Helm by changing the variable would be nicer but I guess that would be an issue with helm and not helmfile.
